scanner/find.go

```diff
@@ -78,23 +93,62 @@ func walkone(ctx context.Context, dir string, config *Config, results chan strin
 }
 
 // Walk finds all git repositories in the directories specified in config
-func Walk(ctx context.Context, config *Config, results chan string) error {
+func Walk(ctx context.Context, config *Config, results chan string, ignore_dir_errors bool) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	completeIncludeList := config.ScanDirs.Include
+
 	var errors errgroup.Group
+
 	for i := range config.ScanDirs.Include {
 		j := i // copy loop variable
+		globPath := config.ScanDirs.Include[j]
+
+		if(string(globPath[len(globPath)-1:]) == "*"){
```

Comment:
I think we should just treat the path as a regular glob .. expand env vars first but then no need to special case this, just use [filepath.Glob](https://pkg.go.dev/path/filepath#Glob).  Expand the globs in a separate function.